.vscode/settings.json

```diff
@@ -1,3 +1,6 @@
 {
-  "eslint.validate": ["json"]
+  "eslint.validate": [
+    "json"
+  ],
+  "cSpell.language": "en,es-ES"
```

Comment:
I prefer to keep this file ignore for now. Everyone has its own extension.

Reply:
it is referenced on the gitIgnore as

`.vscode/*`

and by removing 

`!.vscode/settings.json`

from it.